compiler/AST/expr.cpp

```diff
@@ -6039,55 +6052,51 @@ bool CallExpr::codegenPrimMoveRhsIsSpecialPrimop() {
 
     case PRIM_GET_SVEC_MEMBER_VALUE: {
       /* Get the i'th value from a homogeneous tuple */
-      if (rhs->get(1)->typeInfo()->symbol->hasFlag(FLAG_WIDE_REF)) {
-        // codegenElementPtr/codegenAssign handle wide pointers
-        codegenAssign(get(1),
-                      codegenElementPtr(rhs->get(1),
-                                        codegenExprMinusOne(rhs->get(2))));
+      if (ce->get(1)->typeInfo()->symbol->hasFlag(FLAG_WIDE_REF)) {
+        ret =  codegenElementPtr(ce->get(1), codegenExprMinusOne(ce->get(2)));
 
+        retval = true;
       } else {
-        codegenAssign(get(1),
-                      codegenElementPtr(rhs->get(1),
-                                        codegenExprMinusOne(rhs->get(2))));
+        ret =  codegenElementPtr(ce->get(1), codegenExprMinusOne(ce->get(2)));
+        retval = true;
       }
 
```

Comment:
Moving the assignment retval = true into each branch of the conditional above does not make sense to me. Revert this?